README.md

```diff
@@ -64,6 +64,25 @@ Flavors can be built in parallel easily with the `build.parallel` target:
 make FLAVORS="<flavor> <other flavor> <...>" build.parallel
 ```
 
+### Building with a specific version of Ceph
+Some distributions can select a specific version of Ceph to install.
+You just have to pad the required version to the ceph release code name.
```

Comment:
"pad" is not an accurate word here. Suggest: "append"

Reply:
Not just the version but also the package-manager-appropriate version specifier/separator '-' or '=' must also be part of what is appended.